syntax = "proto3";

package credentials;

service CredentialService {
  rpc StoreCredential(CredentialRequest) returns (CredentialResponse);
}

message CredentialRequest {
  string user_id = 1;
  bytes aaguid = 2;
  bytes credential_id = 3;
  bytes public_key = 4;
}

message CredentialResponse {
  string message = 1;
}
